pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">		 
+	<modelVersion>4.0.0</modelVersion>
+    <groupId>com.cpal</groupId>
+    <artifactId>coding-solutions</artifactId>
+    <version>1.0</version>
+    <packaging>pom</packaging>
+	<parent>
```

Comment:
remove this parent

Reply:
here we need the parent definition in the coding-solutions, this cant be moved to Tax-calculator, since child cant have 2 parent definitions.